src/Command/SonataListFormMappingCommand.php

```diff
@@ -17,6 +17,9 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\HttpKernel\Kernel;
 
+/**
+ * @deprecated the form mapping feature should be disabled
```

Comment:
`deprecated since...` please follow the format described on the contributing guide.